contexts/arbo-context.tsx

```diff
@@ -105,14 +106,21 @@ export const arboReducer = (state: ArboStateType, action: ArboAction) => {
         ...state,
         filteredData: filterData(action.payload.data, selectedFilters),
         selectedFilters: selectedFilters,
-        dataFiltered: true
+        dataFiltered: true,
       };
     case ArboActionType.INITIAL_DATA_FETCH:
       return {
         ...state,
         filteredData: action.payload.data,
         selectedFilters: initialState.selectedFilters,
-        dataFiltered: false
+        dataFiltered: false,
+      };
+    case ArboActionType.RESET_FILTERS:
+      return {
+        ...state,
+        filteredData: [],
```

Comment:
I do not want to hardcode this logic here. Instead we should call the filterData function but have the selected filters be an empty list.

In fact we do not NEED another dispatch action as this basically does what filtered data would do if we manually cleared all filters, so let us keep that logic consistent

Reply:
I have changed this to call the filterData function as mentioned above it will now be as follows (in the next push):
 case ArboActionType.RESET_FILTERS:
      return {
        ...state,
        filteredData: filterData(action.payload.data, {}),
        selectedFilters: initialState.selectedFilters,
        dataFiltered: false,
      };